src/test/cpp/throughput/throughput-main.cpp

```diff
@@ -71,14 +74,14 @@ static void benchmark_conversion_pattern( std::string name,
 		delta_d,
 		int(howmany / delta_d) );
 
-	results.push_back( howmany / delta_d );
+	results.push_back( uint64_t(howmany / delta_d) );
```

Comment:
If `results` use `uint64_t`, shouldn't the line above use that type as well? How about the following for those two places instead, which prevents double calculation at the same time.

```cpp
	results.push_back( uint64_t(howmany / delta_d) );
	LOG4CXX_INFO_FMT( console, "Log4cxx {} pattern: {} Elapsed: {:.4} secs {:L}/sec",
		name,
		conversion_pattern,
		delta_d,
		results.back());
```

[ghpr_92.diff.txt](https://github.com/apache/logging-log4cxx/files/7787940/ghpr_92.diff.txt)